statick_tool/plugins/tool/clang_format_tool_plugin.py

```diff
@@ -59,7 +59,7 @@ def scan(self, package, level):
             output = subprocess.check_output([clang_format_bin,
                                               "--dump-config"],
                                              stderr=subprocess.STDOUT)
-            format_file_name = self.plugin_context.resources.get_file(".clang-format")
+            format_file_name = self.plugin_context.resources.get_file("_clang-format")
```

Comment:
Why do you want to switch from `.clang-format` to `_clang-format`? They appear to still have the same contents. I like `.clang-format` because you can copy it straight into your home directory and apply it to files and know that they will pass the statick check.

Reply:
That's a holdover from the changes I made originally, I seem to recall having some issue with .clang_format vs _clang_format but I don't remember the specifics. I'll remove that change and see if it still works.

Reply:
Okay. If it doesn't work just put that change back and I'll open a new issue to investigate.